src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -111,7 +111,7 @@ class Tmux(_Ssh):
     arguments = (
         Argument('server', help="""
             Server to run tmux session on.
-            Use '-' to for default (webworkers:0)
+            Use '-' to for default (django_manage:0, which depends on env)
```

Comment:
I think you can just have this say `Use '-' for default (django_manage:0)`, fixing the typo in the original. (Remember to run `make` again!)